dan_lombardy/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Your logs folder should be ignored. Usually you don't want to include files your program creates in your repo.